docs/query.rst

```diff
@@ -282,3 +282,19 @@ You can view full example here:  :ref:`example_prefetching`
 
 .. autoclass:: tortoise.query_utils.Prefetch
     :members:
+
+
+F expression
+================
+
+An F() object represents the value of a model field. It makes it possible to refer to model field values and perform database operations using them without actually having to pull them out of the database into Python memory.
+
+For example to use ``F`` to update user balance atomic:
+
+.. code-block:: python3
+
+    from tortoise.expressions import F
+    await User.filter(id=1).update(balance = F('balance') - 10)
+    await User.filter(id=1).update(balance = F('balance') + F('award'), award = 0)
+
+But ``user.balance = F('balance') - 10`` is not supported now.
```

Comment:
I think this last line may confuse users?
May be better to state we ONLY support `F` expressions as part of a `.update()` queryset parameter for now?

Reply:
ok,I will update docs and fix the error test later.